step.yml

```diff
@@ -196,6 +196,16 @@ inputs:
       value_options:
         - "false"
         - "true"
+  - bundletool_version: "0.13.4"
```

Comment:
Agreed on adding a fixed version, instead of parsing the possible latest, to prevent unforeseen changes that cause breakage.